addons/website_forum/static/src/js/website_forum.js

```diff
@@ -315,22 +318,28 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                 } else if (data.error === 'post_non_flaggable') {
                     message = _t("This post can not be flagged");
                 }
-                this.displayNotification({
+                self.displayNotification({
```

Comment:
use this and es6? `data =>`

Reply:
@JKE-be Actually this line should be fixed with your PR (this is the bug we discussed yesterday introduced by https://github.com/odoo/odoo/commit/57ba855b66f2ff1d393f639bbc18dd444e2afa8f)
So this diff should be removed from this PR once bug is fixed